src/js/BookReader.js

```diff
@@ -4133,17 +3782,7 @@ BookReader.prototype.reloadImages = function() {
 };
 
 /**
- * @return {number} (in pixels) of the toolbar height. 0 if no toolbar.
- */
-BookReader.prototype.getToolBarHeight = function() {
-  if (this.refs.$BRtoolbar && this.refs.$BRtoolbar.css('display') === 'block') {
-    return (this.refs.$BRtoolbar.outerHeight() + parseInt(this.refs.$BRtoolbar.css('top')));
-  } else {
-    return 0;
-  }
-}
-
-/**
+ * @todo Move to Navbar.js
```

Comment:
if we won't do this now, can we make an issue for it to do later?

Reply:
Actually, I think I'll just remove the comment; I'm a little on the fence about moving the method; I'll let future devs decide :+1: